subcommands/keys/cmd.go

```diff
@@ -27,8 +27,17 @@ to own the PKI infrastructure of their factory. This infrastructure is used
 to manage mutual TLS between your devices and the Foundries.io device gateway.`,
 }
 
+var tufCmd = &cobra.Command{
+	Use:   "tuf",
+	Short: "Manage The Update Framework Keys for your factory",
+	Long: `Every factory uses TUF to provision OTA updates to devices.
+This command allows customers to own TUF private keys of their factory in order to
+warrant that only customer's personnel can crypto-sign device software updates.`,
```

Comment:
It might be easier to just drop the `Long` attribute so we don't get into a grammatical hole. But perhaps we could state it this way:
```
These sub-commands allow you to manage your Factory's TUF private keys and ensure you are in complete control of your OTA metadata.
```

Actually - i don't think there's a good short answer. Each subcommand's message does a decent job explaining what/why. So leaving this out might be best.

Reply:
Actually, I like your suggested help message (only changed two words).
We need _something_ here; otherwise `keys tuf` looks poor compared to `keys ca`.

fixed in the latest force-push